README.md

```diff
@@ -40,10 +40,17 @@ import parseSwaggerDocumentation from 'api-doc-parser/lib/swagger/parseSwaggerDo
 parseSwaggerDocumentation('https://demo.api-platform.com/docs.json').then(({api}) => console.log(api));
 ```
 
+**OpenApi v3**
```

Comment:
I think we can promote it over Swagger

Reply:
Do you mean we should remove everything related to Swagger in the Readme?